ui/src/test/AuditView/__snapshots__/audit-rules.test.tsx.snap

```diff
@@ -0,0 +1,70 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`AuditHosts should render the AuditView component 1`] = `
```

Comment:
When we started rewriting Controller we used snapshots in our tests.  However, not long after we removed them and abandoned them because they failed all the time, because we were developing and changing the views so quickly.  Because they failed so frequently we all ignored them, which prompted us to remove them. If this is the case for EDA I recommend removing them.

Reply:
Snapshots are only as the first rounds of tests for the lists. Was planning to replace them with more detailed tests later. This no longer applies with the move to monorepo.